src/utils/helpers.js

```diff
@@ -77,6 +77,83 @@ var helpers = {
         return [hrz_altitude, hrz_azimuth]
     },
 
+    siderealTimetimesens: function(lon, date) {
```

Comment:
It would help to specify what type `date` is. I'm not sure if I should pass in a string with some format, or a moment object, etc. 

Also hope I don't sound like a broken record, but I do not understand the name of this function 😅

https://images.techhive.com/images/idge/imported/article/itw/2013/10/23/programmers_hardest_tasks-600x700-100521914-orig.jpg